server/main/templates/main/study-lines.html

```diff
@@ -55,6 +55,7 @@
     </p>
   </div>
   {% endif %}
+  <h1>{% block body_title %}{{ block.super }}{% endblock body_title %}
```

Comment:
Ditto here. The `body_title` `block` is defined in the base template, it should not be repeated here unless actually replacing its content.